Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# reference: https://stackoverflow.com/a/51683309/3711475
```

Comment:
Probably better to reference the official docs, https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker

Reply:
Probably will add both, since it is a mixture from both places.

Reply:
done

Reply:
done